Test-network.md

```diff
@@ -140,23 +140,59 @@ $ storj-sim network run
 ### Using Redis
 A few different processes can use redis as their database. Live accounting cache and revocation db both need to use redis on the Satellite. They will exist as separate databases on the same redis instance. For example, live accouning may use `redis://127.0.0.1:6379?db=0` while revocation may use `redis://127.0.0.1:6379?db=1`. 
 
-The default on storj-sim is to create a test redis server automatically and assign databases to processes incrementally based on how many Satellites are run. For example, if you're running 2 satellites, a configuration like the following may be generated.
+By default, storj-sim will start a redis server for each Satellite that is run and write the respective connection string for each database to a redis.conf file. You must have `redis-server` installed locally for this process to run. You can download redis at https://redis.io/download. (On macOS you can install redis with homebrew `brew install redis`)
```

Comment:
We should specify which versions of Redis are supported.

Reply:
I think redis 4.0+ should be sufficient. I was using the latest 5.0 but we use 4.0 in gcp